src/node-capnp/capnp.cc

```diff
@@ -1738,6 +1771,24 @@ v8::Handle<v8::Value> toBytes(const v8::Arguments& args) {
   });
 }
 
+v8::Handle<v8::Value> toBytesPacked(const v8::Arguments& args) {
+  // toBytes(builder) -> buffer
+
+  KJV8_UNWRAP(StructBuilder, builder, args[0]);
+
+  return liftKj([&]() -> v8::Handle<v8::Value> {
+    auto unpackedSize = computeSerializedSizeInWords(builder.message) * sizeof(capnp::word);
+    // Packed encoding can add 2 bytes of overhead per 2k segment reached.
+    auto packedOverhead = (((unpackedSize - 1) / 2048) + 1) * 2;
```

Comment:
Hmm, `((unpackedSize + 2047) / 2048) * 2` would look less like it has an underflow bug. :) (There's no actual underflow since the size is never zero, but...)